playbooks/ci-ceph_swift/tasks/pre-deploy.yml

```diff
@@ -0,0 +1,89 @@
+---
+- name: controller 0 hosts file
+  hosts: controller[0]
+  vars_files:
+  - ../vars/main.yml
+  tasks:
+    - name: add controller node to /etc/hosts
+      lineinfile: dest=/etc/hosts regexp={{ testenv_instance_names[0] }}
+                  insertafter=EOF
+                  line="127.0.1.1 {{ testenv_instance_names[0] }}"
+
+- name: controller 1 hosts file
+  hosts: controller[1]
+  vars_files:
+  - ../vars/main.yml
+  tasks:
+    - name: add controller node to /etc/hosts
+      lineinfile: dest=/etc/hosts regexp={{ testenv_instance_names[1] }}
+                  insertafter=EOF
+                  line="127.0.1.1 {{ testenv_instance_names[1] }}"
+
+- name: compute 0 hosts file
+  hosts: compute[0]
+  vars_files:
+  - ../vars/main.yml
+  tasks:
+    - name: add compute node to /etc/hosts
+      lineinfile: dest=/etc/hosts regexp={{ testenv_instance_names[2] }}
+                  insertafter=EOF
+                  line="127.0.1.1 {{ testenv_instance_names[2] }}"
```

Comment:
These seem pretty static and hardcoded, and thus fragile if we wanted to expand the group members.